### ISSUE TITLE
Hi, I added some third party libraries in iOS and Android but that library was coming under MIT Licence can i add Library in my project or i want to pay anything to MIT LIicence or is it free.

### ISSUE THREAD

**Issue by SravanZetamp** [angular/angular.js#16701]
**I'm submitting a ...**

- [ ] bug report
- [ ] feature request
- [x] other 

**Current behavior:**

**Expected / new behavior:**

**Minimal reproduction of the problem with instructions:**

**AngularJS version:** 1.x.y

**Browser:** [all | Chrome XX | Firefox XX | Edge XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Anything else:**

**Comment by petebacondarwin** [comment]
@SravanZetamp - you can see the MIT license here https://github.com/angular/angular.js/blob/master/LICENSE. GitHub also kindly adds some summary information about its use.

tl;dr you don't have to pay for MIT licensed software.
